src/stripeDotnetLanguageServer/Program.cs

```diff
@@ -0,0 +1,99 @@
+                        .MinimumLevel.Debug()
+                        .CreateLogger();
+
+            Log.Debug("Creating project..");
+            // TODO -- Get this as an arg from the client.
+            const string projectFile = "/Users/gracegoo/stripe/samples/accept-a-payment-dotnet/server/server.csproj";
+
+            // Without this MSBuild can't find the SDK folder.
+            MSBuildLocator.RegisterDefaults();
+            var workspace = MSBuildWorkspace.Create();
+
+            // We need to register to the failed event to be notified if there were any failures.
+            // https://docs.microsoft.com/en-us/dotnet/standard/events/
+            workspace.WorkspaceFailed += WorkspaceFailed;
+            var project = await workspace.OpenProjectAsync(projectFile);
+
+            Log.Debug("Created project...");
+
+            // Check workspace
+            foreach (Document document in project.Documents)
+                Log.Debug("Document in project " + document.Name);
+
+            Log.Debug("Starting language server from the VSCode repo!...");
+            var server = await OmniSharp.Extensions.LanguageServer.Server.LanguageServer.From(options =>
+                options
+                    .WithInput(Console.OpenStandardInput())
+                    .WithOutput(Console.OpenStandardOutput())
+                    .ConfigureLogging(
+                            x => x
+                                .AddSerilog(Log.Logger)
+                                .AddLanguageProtocolLogging()
+                                .SetMinimumLevel(LogLevel.Debug)
+                        )
+                    // .WithHandler<TextDocumentHandler>()
+                    .WithHandler<APIReferenceHandler>()
+                    .WithServices(x => x.AddLogging(b => b.SetMinimumLevel(LogLevel.Trace)))
+                    .WithServices(
+                        services =>
+                        {
+                            services.AddSingleton(
+                                provider =>
+                                {
+                                    var loggerFactory = provider.GetService<ILoggerFactory>();
+                                    var logger = loggerFactory.CreateLogger<WorkspaceManager>();
+                                    logger.LogInformation("Configuring");
+                                    return new WorkspaceManager(logger, workspace);
+                                }
+                            );
+                            services.AddSingleton(
+                                new ConfigurationItem
+                                {
+                                    Section = "typescript",
+                                }
+                            ).AddSingleton(
+                                new ConfigurationItem
+                                {
+                                    Section = "terminal",
+                                }
+                            );
```

Comment:
What are these ConfigurationItems used for?

Reply:
Oops, leftovers from the [sample](https://github.com/OmniSharp/csharp-language-server-protocol). I think these are used to send over configs from the client -- let me remove these. Good catch!